defaults/main.yml

```diff
@@ -60,3 +60,49 @@ steamengine_java_base_opts:
 steamengine_java_opts:
   - "-Xmx512M"
   - "-Xms512M"
+
+# DRUPAL
+# Path of drush launcher
+steamengine_drush_launcher_path: "/usr/local/bin/drush"
+# Command run at the end of deployment
+steamengine_drupal_drush_command_deployment:
+  - "deploy -y"
+# List of directory we need write permission
+steamengine_drupal_path_with_write_permission:
```

Comment:
Entry in steamengine_drupal_persistent_directory should give write permisison to the dest folder (and it seems to be the case) because persitence assume writing...

The example put "web/sites/default/files" in both steamengine_drupal_path_with_write_permission and steamengine_drupal_persistent_directory, it should only be specified in steamengine_drupal_persistent_directory. Also the comment should say that every writes in steamengine_drupal_path_with_write_permission will be loss when deploying.

Reply:
Still relevant

Reply:
Closing, usage of the new "steamengine_persistent_directories" fix this issue.